examples/blog-multiple-authors/src/pages/authors/[author].astro

```diff
@@ -11,8 +11,16 @@ let canonicalURL = Astro.request.canonicalURL;
 
 // collection
 import authorData from '../../data/authors.json';
+
+interface Frontmatter {
+  date: number;
+  description: string;
+  title: string;
+  author: string;
+}
+
 export function getStaticPaths() {
-  const allPosts = Astro.fetchContent('../post/*.md');
+  const allPosts = Astro.fetchContent<Frontmatter>('../post/*.md');
```

Comment:
Not sure if you set a default already, but can fetchContent default to `any` or `unknown` by using `<T = ...>`?

Reply:
Yeah probably, I'll test this out.

Reply:
It does default to `unknown` but I think we probably want this to be `any` so you don't get type errors when trying to access a variable.

https://github.com/snowpackjs/astro-language-tools/issues/61

Reply:
yea, I think `any` is a better user experience for now. TS seems to be moving more towards unknown (for example, `catch (err)` is now `err: unknown`) but we're trying to strike more of a "ts is optional and opt-in" line, so no errors by default feels right to me.